### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by mirpedrol**
Rich-codex images for docs are now generated on the website repo

### CODE REVIEW

**Comment by mirpedrol** [comment]
@nf-core-bot changelog Remove `rich-codex.yml` action, images are now generated on the website repo

**Review by mashehu — APPROVED** [review]
LGTM if we don't put the docs back to this repo

**Comment by mirpedrol** [comment]
> LGTM if we don't put the docs back to this repo

Let's wait before merging this PR :)

**Review by mashehu — APPROVED** [review]

**Pull request merged**
